### PR TITLE
PROJ-191 Link opportunity

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added dynamic link for either opportunity or project

Note: this relies on the fact that the insightlyProjectId contains the opportunity id
---
Ticket: https://jira.uitdatabank.be/browse/PROJ-191

### CODE REVIEW

File: app/scripts/factories/project.factory.js [hunk 1]
```diff
@@ -47,6 +47,11 @@ function cultuurnetProjectFactory(IntegrationType, ProjectStatuses) {
       this.sapiVersion = jsonObject.sapiVersion || '2';
       this.contentFilter = jsonObject.contentFilter || '';
 
+      this.insightlyUrl = 'https://crm.na1.insightly.com/details/Project/' + this.insightlyProjectId;
+      if (jsonObject.opportunityIdInsightly && !jsonObject.projectIdInsightly) {
+        this.insightlyUrl = 'https://crm.na1.insightly.com/details/Opportunity/' + this.insightlyProjectId;
```
Comment (bertramakers) [inline app/scripts/factories/project.factory.js:52]: Should this not use the `opportunityIdInsightly` instead at the end of the URL? Why do we use the `insightlyProjectId` if we have a `opportunityIdInsightly`?

Comment (LucWollants) [reply app/scripts/factories/project.factory.js:52]: That is indeed more safe for the future. But for now the `insightlyProjectId` will be the `opportunityIdInsightly` if it's just an Opportunity inside Insightly.

Comment (LucWollants) [reply app/scripts/factories/project.factory.js]: I made the code a bit more explicit and future proof.

Comment (bertramakers) [reply app/scripts/factories/project.factory.js]: Thanks! Is it possible that there's an actual bug now? 😄 This links to an opportunity using the new project id if I'm not mistaken https://github.com/cultuurnet/projectaanvraag-angular/pull/59/files#diff-a476b2a743a63ecd3e017f601e9b25425825df080464fc66c33e15da93730f08R52

Comment (LucWollants) [reply app/scripts/factories/project.factory.js]: Indeed, I made a copy/paste error here. Should be fine now 😅

**Review by bertramakers — APPROVED** [review]

File: app/scripts/factories/project.factory.js (hunk 1 above)
Comment (bertramakers) [reply app/scripts/factories/project.factory.js]: Thanks for the quick fix!

**Pull request merged**